### PR TITLE
Support direct image/file url

### PR DESCRIPTION
**Pull request by AliceYuan**
#82

### CODE REVIEW

**Comment by AliceYuan** [comment]
Still need to write unit tests! Also still need to support direct files

File: client/url_utils.py [hunk 1]
```diff
@@ -23,13 +23,9 @@ def get_redirect_url(head):
...
 
-def get_url_response_code(head):
-    status_code = None
-    if head is None:
-        status_code = StatusCode.BAD_REQUEST
-    else:
-        status_code = head.status_code
-    return status_code
+def get_content_type(head):
+    content_type = head.headers['content-type']
```
Comment (MeenuV) [inline client/url_utils.py]: Smart! :dancer:

**Comment by MeenuV** [comment]
LGTM. Up to you if you wanna add tests before merging

**Comment by AliceYuan** [comment]
I'm adding tests after merge of #92

**Pull request merged**